media/css/pebbles/components/_global-nav.scss

```diff
@@ -515,6 +531,7 @@ html.moz-nav-open .moz-global-nav-page-mask {
 
         .download-title {
             font-weight: bold;
+            white-space: nowrap;
```

Comment:
I see this here in the scss version, but not the less version? Should they both be the same?